addons/website_product_comparison/views/product_comparison_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="product_attribute_category_tree_view" model="ir.ui.view">
+        <field name="name">product.attribute.category.tree</field>
+        <field name="model">product.attribute.category</field>
+        <field name="arch" type="xml">
+            <tree string="Product Attribute Category" editable="top">
+                <field name="sequence"/>
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="product_attribute_category_action" model="ir.actions.act_window">
+        <field name="name">Website attribute categories</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">product.attribute.category</field>
+        <field name="view_mode">tree</field>
+        <field name="view_type">form</field>
+    </record>
+
+    <menuitem action="product_attribute_category_action"
+        id="menu_attribute_category_action"
+        parent="website_sale.menu_ecommerce_settings" sequence="11"/>
```

Comment:
Is this model strictly related to the ecommerce? I don't think so. `sale.prod_config_main` is maybe better (could be both though) and a different action name